src/HeapOrdering.sol

```diff
@@ -58,8 +58,10 @@ library HeapOrdering {
 
     /// PRIVATE ///
 
-    /// @notice Computes a new suitable size from `_size` that is smaller than `_maxSortedUsers`.
-    /// @dev We use division by 2 to remove the leaves of the heap.
+    /// @notice Computes a new suitable size for the Heap from `_size` that is smaller than `_maxSortedUsers`.
+    /// @notice It helps solving to have all the liquity on the same path.
+    /// @notice You can go to the README file for more information.
```

Comment:
```suggestion
    /// @notice Computing the size this way is meant to avoid having all the liquidity in the same path.
```
The code should not depend on the README in my opinion (it might not even be available to the reader).